### PR TITLE
Add redirect when the user submit one solution

### PR DESCRIPTION
**Pull request by boaglio15**

### CODE REVIEW

File: client/src/actions/proposition/propositionsActions.js [hunk 1]
```diff
@@ -1,49 +1,58 @@
...
-      let username = getState().user.currentUser.username;
-      axios.post('http://localhost:55555/users/'+idUsr+'/challenge/'+idChallenge+'/propsitions', null, {
-        headers: {
-          Authorization: "Basic" + base64.encode(username + ":" +localStorage.getItem("password"))
+    dispatch(fetchPropositionsRequest());
+    let base64 = require("base-64");
+    let username = getState().user.currentUser.username;
+    axios
+      .post(
+        "http://localhost:55555/users/" +
```
Comment (se-unrc) [inline client/src/actions/proposition/propositionsActions.js]: instead localhost could we do another commit to use `process.env.REACT_APP_API_HOST` and you can configure it when needed using the `.env` file

File: client/src/actions/proposition/propositionsActions.js [hunk 2]
```diff
@@ -1,49 +1,58 @@
...
 export const fetchPropositionsGame = (idUsr, idChallenge) => {
   return async function(dispatch, getState) {
-    if (getState().propositions.data.length === 0) {
-      dispatch(fetchPropositionsRequest())
-      let username = getState().user.currentUser.username;
-      axios.post('http://localhost:55555/users/'+idUsr+'/challenge/'+idChallenge+'/propsitions', null, {
-        headers: {
-          Authorization: "Basic" + base64.encode(username + ":" +localStorage.getItem("password"))
+    dispatch(fetchPropositionsRequest());
+    let base64 = require("base-64");
```
Comment (se-unrc) [inline client/src/actions/proposition/propositionsActions.js]: instead require it here, we should use the import method. So, instead 
```
let base64 = require("base-64");
```

at the top of the file you can use 
```
import base64 from 'base-64'
```

File: client/src/actions/proposition/propositionsActions.js [hunk 3]
```diff
@@ -1,49 +1,58 @@
...
-          dispatch(fetchPropositionsSucess(res))
-        })
-        .catch(error => {
-          console.log(error)
-          dispatch(fetchPropositionsFailure(error.message))
-        })
-    }
-  }
+      .catch(error => {
+        console.log(error);
```
Comment (se-unrc) [inline client/src/actions/proposition/propositionsActions.js]: get rid of logs please

File: client/src/actions/proposition/propositionsActions.js (hunk 1 above)
Comment (boaglio15) [reply client/src/actions/proposition/propositionsActions.js]: Ready

File: client/src/actions/proposition/propositionsActions.js (hunk 2 above)
Comment (boaglio15) [reply client/src/actions/proposition/propositionsActions.js]: Ready

File: client/src/actions/proposition/propositionsActions.js (hunk 3 above)
Comment (boaglio15) [reply client/src/actions/proposition/propositionsActions.js]: Ready
